libs/hpc-ui/src/lib/styling.tsx

```diff
@@ -49,6 +49,15 @@ body {
   flex-grow: 1;
 }
 
+.${CLASSES.VISUALLY_HIDDEN} {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
```

Comment:
I've made lots of comments on formatting on these and that isn't something a review should be concerned with. The big downside with the idea of Tailwind (with CSS-in-JS) is that we don't get formatting with the tooling we're using.

In this file, all the CSS statements are having a space after colon, but you didn't use it
```suggestion
.${CLASSES.VISUALLY_HIDDEN} {
  position: absolute;
  left: -10000px;
  top: auto;
  width: 1px;
  height: 1px;
  overflow: hidden;
}
```